Add new PUT method for load balancers: /{id}/setAttributes

mgulimonov opened the pull request:
Ref https://github.com/TheWeatherCompany/grid-api/issues/871

Commit:
add to aws LB modifyLoadBalancerAttributes (fixes #871)

Commit:
change int to Integer (fixes #871)

Commit:
change boolean to Boolean (fixes #871)

Commit:
Merge branch 'develop' into issue-871

clstokes commented:
Ready when corresponding changes from https://github.com/TheWeatherCompany/dasein-cloud-core/pull/81#discussion_r13948512 are complete.

Commit:
add getLoadBalancerAttributes (fixes #871)
Options moved to AttributesOptions class

Commit:
Merge branch 'develop' into issue-871

Commit:
Merge branch 'develop' into issue-871

eyaroslavtsev committed:
Add 'idleConnectionTimeout' parameter to load balancer (#871 #1036)

eyaroslavtsev committed:
Merge branch 'develop' into issue-871

clstokes committed:
Corresponding rename changes for...

https://github.com/TheWeatherCompany/dasein-cloud-core/commit/d6f142778f21b58a3dc4426ce8d9573a7903a10a
